Eatery/Controllers/Eateries/CampusEateryMealTableViewController.swift

```diff
@@ -25,21 +26,13 @@ class CampusEateryMealTableViewController: UITableViewController {
             // don't know the menu
             menu = nil
```

Comment:
don't think we need this b/c `menu` is default `nil` right?

Reply:
Swift will complain that `menu` is uninitialized otherwise